src/js/consts/dataFormat/graphConfig.jsx

```diff
@@ -67,7 +67,20 @@ function loadGraphColors(payload) {
 
       payload.data.labels.forEach((label, index) => {
         const labelColor = Object.keys(payload.config.colors.labels)
-          .find(key => payload.config.colors.labels[key].includes(label));
+          .find((key) => {
+            let response = false;
+            payload.config.colors.labels[key].forEach((labelConfig) => {
```

Comment:
What's the point of this part? And when the line 80 will need to be removed?

Reply:
This is because we have to get the code of each labelConfig to compare it to the label.code. In fact, now, we don't send a string from the config for the color configuration but also a map

Reply:
@awalkowiak  it will be removed when all the color configurations will sending a map of code and message.